torch/csrc/jit/python/init.cpp

```diff
@@ -858,7 +858,7 @@ void initJITBindings(PyObject* module) {
         tracer::WithNestedTracingFrame env_guard;
 
         // Run the user-supplied function
-        py_func_output = f(*args_tup);
+        py_func_output = f(*args_tup, **kwargs);
```

Comment:
Don't think this code path is currently being tested

Reply:
Yea, you're right.  It's covered with my latest test.